Sources/PointFree/About.swift

```diff
@@ -10,32 +10,26 @@ import Tuple
 
 let aboutResponse =
   writeStatus(.ok)
-    >-> respond(aboutView.contramap(lower))
+    >-> map(lower)
+    >>> respond(
+      view: aboutView,
+      layoutData: { currentUser in
+        SimplePageLayoutData(currentUser: currentUser, data: unit, title: "About Us")
+    }
```

Comment:
here's how you use it. note that the layout needs the current user, but the view only needs `Unit`